workflow/envs/notramp.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - simakro
```

Comment:
you have your own conda channel? :)

Reply:
I have now. Fastest way to make it available and quite convenient. Going to deposit it in bioconda as well, but it is done in another way and I have to read the docs first, because they have a different procedure (Apparently you have to fork the bioconda-recipe repository and then some...)